lib/browser_collection.js

```diff
@@ -26,16 +26,12 @@ class BrowserCollection {
     return this.browsers.find((browser) => browser.id === browserId) || null
   }
 
```

Comment:
Please note this new API in the description.

Reply:
Fixed